GiniVisionExample/GiniVisionExample/AppCoordinator.swift

```diff
@@ -34,18 +38,39 @@ final class AppCoordinator: Coordinator {
         window.rootViewController = rootViewController
         window.makeKeyAndVisible()
     }
+    
+    @discardableResult func showResultsViewController()
+        -> ResultsViewController {
+            resultViewController = ResultsViewController(model: ResultsViewModel(documentService: documentService))
+            resultViewController!.delegate = self
+            rootViewController.present(resultViewController!, animated: true, completion: nil)
+            return resultViewController!
+    }
+}
+// TODO: Move to screen api coordinator when implemented
+extension AppCoordinator: ResultsViewControllerDelegate {
+    func results(viewController: ResultsViewController, didTapDone: ()) {
+        resultViewController?.dismiss(animated: true, completion: nil)
+    }
 }
 
 // MARK: MainViewControllerDelegate
 
 extension AppCoordinator: MainViewControllerDelegate {
     
     func main(viewController: MainViewController, didTapStartAnalysis: ()) {
-        
+        let ibanExtraction = GINIExtraction(name: "iban", value: "DE 1234 5678 9123 4567", entity: "entity", box: [:])
+        let paymentRecipientExtraction = GINIExtraction(name: "paymentRecipient",
+                                                        value: "Rick Sanchez", entity: "entity", box: [:])
```

Comment:
Nice that Rick is using the app, but why do you have his hardcoded data into the production code?

PICKLE RICK!

Reply:
Just for testing the UI and give and example to Ben (he will give a better design to these screens). This and the rest of hardcode data will be removed in a future PR.

Reply:
Well, just make some screenshots for Ben then?

Reply:
I uploaded a version to Hockey so he could test in his phone. As I said, this hardcoded data will be remove in upcoming PRs.